options/internal/gcc/guard-abi.cpp

```diff
@@ -31,6 +31,7 @@ struct Guard {
 	// the first byte's meaning is fixed by the ABI.
 	// it indicates whether initialization has already been completed.
 	uint8_t complete;
+	uint8_t padding[3];
```

Comment:
nit: can you add a comment here? Makes it slightly less confusing to read

Reply:
Added.